sphinxcontrib/spelling/builder.py

```diff
@@ -61,15 +61,35 @@ def init(self):
             f.append(filters.ImportableModuleFilter)
         f.extend(self.config.spelling_filters)
 
-        project_words = os.path.join(self.srcdir,
-                                     self.config.spelling_word_list_filename)
-        self.info('Looking for custom word list in {}'.format(
-            project_words))
+        if not os.path.isdir(self.outdir):
+            os.mkdir(self.outdir)
+
+        #
+        # In case the user has multiple word lists, we combine them into one
+        # large list that we pass on to the checker.
+        #
+        word_list = self.config.spelling_word_list_filename
+        # If we have a list, the combined list is the first list plus all words
+        # from the other lists. Otherwise, word_list is assumed to just be a
+        # string.
+        if isinstance(word_list, list):
+            combined_word_list = os.path.join(self.outdir,
+                                              'spelling_wordlist.txt')
+            # Just append every file to the combined file
+            with open(combined_word_list, 'w') as outfile:
+                for word_file in word_list:
+                    long_word_file = os.path.join(self.srcdir, word_file)
+                    self.info('Looking for custom word list in {}'.format(
+                        long_word_file))
+                    with open(long_word_file) as infile:
```

Comment:
Does this need to handle unicode?

Reply:
It probably does have to handle it, yes. Then I can probably reuse the `codecs` code that was already present in the file.